spec/system/bitklavier_pdc_describe_indexing_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+describe 'PDC Describe research data indexing -- Bitklavier', type: :system do
+  subject(:result) { indexer.map_record(record) }
+  let(:indexer) do
+    Traject::Indexer::NokogiriIndexer.new.tap do |i|
+      i.load_config_file(Rails.root.join('lib', 'traject', 'pdc_describe_indexing_config.rb'))
+    end
+  end
+  let(:pdc_describe_resource_xml) { File.join(fixture_path, 'pdc_describe', 'bitklavier_binaural.xml') }
```

Comment:
I'm curious why you are utilizing XML here and json in the test below.  Are we expecting XML and json formats from PDC Describe, or is one from Dataspace and one from PDC Describe?  This is indicating that it is a pdc describe resource though by the name.

Reply:
PDC Describe returns the JSON resource for indexing and then I convert it to XML for indexing. This let me copy the traject indexing we used for DataSpace indexing, which I thought would make it easier to track when we've indexed all of the fields.